src/nvim/ex_cmds2.c

```diff
@@ -392,12 +392,12 @@ int dbg_check_skipped(exarg_T *eap)
  * This is a grow-array of structs.
  */
 struct debuggy {
-  int dbg_nr;                   /* breakpoint number */
```

Comment:
I think its fine, because this keeps it consistent with the struct that was needed to change.